_data/sites.json

```
@@ -8181,6 +8181,16 @@
         ]
     },
 
+    {
+        "name": "Wedbio.com",
+        "url": "https://wedbio.com",
+        "difficulty": "hard",
+        "notes": "You would need to contact the team by submit contact us form to get your account closed.",
```

```suggestion
        "notes": "You have to contact them via the \"contact us form\" to get your account closed.",
```
Just a suggestion of a rewording. Do you have the link that goes directly to the contact form? That would be very useful!

---

it just use an anchor like this `https://wedbio.com/#contact`, btw how do you add suggestion, I can't edit my pull request anymore via github web.

---

nevermind, have found it. thanks for your suggestion @tupaschoal, please help to review again :)